desktop/webpack.config.base.js

```diff
@@ -15,6 +15,12 @@ console.warn('Injecting defines: ', defines)
 module.exports = {
   module: {
     loaders: [{
+      test: /\.flow?$/,
```

Comment:
you mean the resolve extensions? i think we want that still to help with eslint? i'm hesitant to remove it